packages/cdc-react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,54 @@
+import { ButtonIconPositionTypes, ButtonVariationsTypes } from "src/@types";
+import { Button } from "../Button/Button";
+import "./Card.scss";
+interface CardProps {
+  layout: "horizontal" | "vertical-left" | "vertical-right";
+  action: string;
+  mediaSource?: string;
+  header?: string;
+  sectionTitle?: string;
+  showTitleTop?: boolean;
+  actionButtonVariation?: ButtonVariationsTypes;
+  actionButtonText?: string;
+  actionButtonIcon?: React.ReactNode;
+  actionButtonIconPosition?: ButtonIconPositionTypes;
+  children: React.ReactNode;
+}
+
+export const Card = (props: CardProps) => {
+  return (
+    <div className="card-wrapper">
+      <div className="card-content">
+        {props.header && <div className="card-header">{props.header}</div>}
+        <div className={`card-body ${props.layout}`}>
+          {props.layout === "horizontal" && props.showTitleTop && (
+            <div className="card-section-title">{props.sectionTitle}</div>
+          )}
+          <div
+            className={`card-section-media ${
+              props.showTitleTop ? "" : "media-fill"
+            }`}>
+            {props.mediaSource && (
+              <img src={props.mediaSource} alt={props.sectionTitle} />
+            )}
+          </div>
+          <div className="card-body-content">
+            {!props.showTitleTop && (
+              <div className="card-section-title">{props.sectionTitle}</div>
+            )}
+            <div className="card-section-text">{props.children}</div>
```

Comment:
Let's rename this class to `card-section-body` since there could be other things beside text-based elements.